app/templates/discover.hbs

```diff
@@ -106,8 +106,9 @@
                 </div>
 
                 <div class="m-v-md registration-type-selector">
-                    <i class='type-selector-warning' style='color: red; display: none'>Only available with OSF Registries</i>
-                    <h4 class="filter-heading p-sm">{{t "discover.main.type"}}</h4>
+                    <h4 class="filter-heading p-sm" aria-label={{t "discover.main.osf_registraton_filter"}}>
+                        {{t "discover.main.type"}} <span class="hint--bottom hint--small" data-hint={{t "discover.main.popover.registries.body"}}><i class="fa fa-question-circle permission-info" aria-label={{t "discover.main.popover.registries.body"}}></i></span>
```

Comment:
I tried this initially. However when registration-type-selector is faded out. The tooltip goes below the search results.Please see the attached screen shot.
![image](https://cloud.githubusercontent.com/assets/11130339/23912893/9eb209de-08b7-11e7-9e14-0d8d1747faf0.png)